app/logic/userInsertFunctions.py

```diff
@@ -52,28 +53,45 @@ def createUser(username, student=None, supervisor=None):
 
 def updateDBRecords():
     """
-    This function will update all student and supervisor records according to
+    This function will update all student, supervisor and department records according to
     Tracy data.
     """
     studentsInDB = Student.select()
     supervisorsInDB = Supervisor.select()
+    departmentsFromDB = Department.select()
```

Comment:
Not a blocker and really nitpicky but I think it would be nice to keep naming the same on all of these so it would be departmentsInDB